src/cryptography/x509/__init__.py

```diff
@@ -92,6 +92,9 @@
 OID_STATE_OR_PROVINCE_NAME = NameOID.STATE_OR_PROVINCE_NAME
 OID_SURNAME = NameOID.SURNAME
 OID_TITLE = NameOID.TITLE
+OID_INN = NameOID.INN
+OID_OGRN = NameOID.OGRN
+OID_SNILS = NameOID.SNILS
```

Comment:
@reaperhulk my recollection is that the existing ones are all here for backwards compatibility, and that we don't add new ones here, they can be accessed just via the enum, is that right/

Reply:
ok